docs/reference/upgrade.asciidoc

```diff
@@ -87,6 +87,15 @@ When upgrading to a new version of {es}, you need to upgrade each
 of the products in your Elastic Stack. For more information, see the
 {stack-ref}/upgrading-elastic-stack.html[Elastic Stack Installation and Upgrade Guide].
 
+ifeval::["{release-state}"!="unreleased"]
+[[upgrade-pre-release]]
+NOTE: This documentation is for {es} {version} which is not yet released. You
+may run a pre-release build of {es} for testing, and you may upgrade from an
+earlier released version to a pre-release build of {es} {version}, but
```

Comment:
I think this should be the last released version. Otherwise, this implies you could upgrade from 6.8 to the 8.0 pre-release build.
```suggestion
last released version to a pre-release build of {es} {version}. However,
```

Reply:
:+1: good point, I added "if permitted by the compatibility table above" instead since we do permit folks to upgrade from earlier versions too.